scripts/core/common.yml

```diff
@@ -316,3 +316,12 @@ members:
     - type: uint64_t
       name: depth
       desc: "[in] scalar (scalar)"
+--- #--------------------------------------------------------------------------
+type: typedef
+desc: "File Descriptor type."
+name: $x_file_descriptor_t
+value:
+    - if: Windows
+      value: HANDLE
+    - else: ""
+      value: int
```

Comment:
nit:
```suggestion
      value: uint32_t
```

Reply:
Are you sure? `open()` will return an `int` as a file descriptor. `-1` is often used to indicate an invalid file descriptor.

Reply:
Hmm, that seems sound to me. I stand corrected!